rakelib/55-kitchen.rake

```diff
@@ -5,16 +5,8 @@ task :test_kitchen do
   Kitchen.logger = Kitchen.default_file_logger
   @loader = Kitchen::Loader::YAML.new(project_config: './.kitchen.ec2.yml')
   config = Kitchen::Config.new(loader: @loader)
-  if ENV['KITCHEN_NO_CONCURRENCY']
-    config.instances.each do |instance|
-      instance.test(:always)
-    end
-  else
-    threads = []
-    config.instances.each do |instance|
-      threads << Thread.new { instance.test(:always) }
-    end
-    threads.map(&:join)
+  config.instances.each do |instance|
```

Comment:
I'd rather like to keep the code here like this and use an attribute in the kitchen.ec2.yml to control the concurrency.
and base on this attribute we define if we want concurrency or not?